src/coreclr/hosts/corerun/CMakeLists.txt

```diff
@@ -14,6 +14,7 @@ set(CLR_CMAKE_KEEP_NATIVE_SYMBOLS TRUE)
 add_executable_clr(corerun
   corerun.cpp
   dotenv.cpp
+  ${CLR_SRC_NATIVE_DIR}/minipal/is_native_debugger_present.c
```

Comment:
Wasn't there some effort to make the minipal and static lib?

Reply:
Yes whichever PR is merged first, the other one will need to resolve conflicts as we usually do. :)